src/watchmaker/__init__.py

```diff
@@ -108,30 +135,35 @@ def _get_config_data(self):
                     shutil.copyfileobj(response, outfile)
                 self.config_path = 'config.yaml'
             except urllib.error.URLError:
-                print('The URL used to get the user config.yaml file did not '
-                      'work!\nPlease make sure your config is available.')
+                logging.critical(
+                    'The URL used to get the user config.yaml file did not '
+                    'work!\nPlease make sure your config is available.'
+                )
                 sys.exit(1)
 
         if self.config_path and not os.path.exists(self.config_path):
-            self.logger.warning(
+            logging.warning(
                 'User supplied config {0} does not exist. '
                 'Using the default config.'.format(self.config_path)
             )
             self.config_path = self.default_config
         elif not self.config_path:
-            self.logger.warning(
+            logging.warning(
                 'User did not supply a config.  Using the default config.'
             )
             self.config_path = self.default_config
         else:
-            self.logger.info('User supplied config being used.')
+            logging.info('User supplied config being used.')
         with open(self.config_path) as f:
             data = f.read()
 
         if data:
             self.config = yaml.load(data)
         else:
-            logging.debug('No data to load.')
+            logging.critical(
+                'Unable to load the data of the default or'
+                ' the user supplied config.'
+            )
```

Comment:
I think the only way to get into this branch is if the file is empty. This should probably exit if there is no config data, right? Does `logging.critical()` automatically exit?